sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ConsoleLogging.cs

```diff
@@ -48,7 +48,8 @@ protected sealed override void Initialize(SonarAnalysisContext context)
             context.RegisterSyntaxNodeActionInNonGenerated(
                 c =>
                 {
-                    if (c.IsTest())
+                    if (c.IsTest() ||
+                        c.Compilation.Options.OutputKind == OutputKind.ConsoleApplication)

```

Comment:
We should refactor the _Verifier_ at some point so we can easily write unit tests for this kind of thing (either by changing it so we can pass in _CompilationOptions_ or perhaps a _Compilation_).

Reply:
Yes there is a lot of work todo on the Verifier (see #817)